bugbug/tools/code_review.py

```diff
@@ -663,18 +678,30 @@ def __init__(self, patch_id: str, auth: dict) -> None:
         self.rev_id = int(patch_id)
 
     @cached_property
-    def raw_diff(self) -> str:
+    def _revision_metadata(self) -> dict:
         import swarm
 
-        revisions = swarm.get(self.auth, rev_ids=[self.rev_id], version_l=[0, 1])
+        revisions = swarm.get(self.auth, rev_ids=[self.rev_id])
         assert len(revisions) == 1
 
-        return revisions[0]["fields"]["diff"]
+        return revisions[0]
+
+    @property
+    def raw_diff(self) -> str:
+        revision = self._revision_metadata
+
+        return revision["fields"]["diff"]
 
     @cached_property
     def base_commit_hash(self) -> str:
         raise NotImplementedError
 
+    @property
+    def date_created(self) -> datetime:
+        revision = self._revision_metadata
+
+        return datetime.fromtimestamp(revision["fields"]["created"])
+
```

Comment:
Are you sure about the Swam changes?

Reply:
I checked the API documentation, but if it is incorrect, it will crash; then it will be corrected.